SECURITY.md

```diff
@@ -21,7 +21,7 @@ security features are welcome on GitHub Issues.
 
 Security updates will be released on a regular cadence. Many of our projects
 will typically release security fixes in conjunction with the
-[Oracle Critical Patch Update][3] program. Additional
+Oracle Critical Patch Update program. Additional
```

Comment:
```suggestion
[Oracle Critical Patch Update][4] program. Additional
```

Then change `[security alerts][4]` to `[security alerts][5]` and modify the links at the bottom of the file accordingly:

```markdown
[4]: https://www.oracle.com/security-alerts/#CriticalPatchUpdates
[5]: https://www.oracle.com/au/security-alerts/#SecurityAlerts
```